ml-agents-envs/mlagents_envs/tests/test_rpc_communicator.py

```diff
@@ -28,3 +37,46 @@ def test_rpc_communicator_create_multiple_workers():
     second_comm = RpcCommunicator(worker_id=1)
     first_comm.close()
     second_comm.close()
+
+
+@mock.patch.object(grpc, "server")
+@mock.patch.object(
+    mlagents_envs.rpc_communicator, "UnityToExternalServicerImplementation"
+)
+def test_rpc_communicator_initialize_OK(mock_impl, mock_grpc_server):
```

Comment:
test_rpc_communicator_initialize_OK and test_rpc_communicator_initialize_timeout passed with the old functionality too; test_rpc_communicator_initialize_callback is the new stuff.